test/functional/legacy/082_spec.lua

```diff
@@ -0,0 +1,105 @@
+-- Tests for case-insensitive UTF-8 comparisons (utf_strnicmp() in mbyte.c)
+-- Also test "g~ap".
+
+local helpers = require('test.functional.helpers')
+local feed, insert, source = helpers.feed, helpers.insert, helpers.source
+local clear, execute, expect = helpers.clear, helpers.execute, helpers.expect
+
+describe('82', function()
```

Comment:
Same for the file name, could add a descriptive phrase in the test name as has been done for the other tests, `082_string_comparison_spec.lua`.